practical-07/src/store/actions.js

```diff
@@ -0,0 +1,98 @@
+import axios from "axios";
+import route from "../route/route";
+
+export default {
+    fetchData({ commit }) {
+        axios
+            .get("https://testapi.io/api/dartya/resource/cardata")
```

Comment:
Codacy has a fix for the issue: [Delete `······`](https://app.codacy.com/gh/mansinakrani/vuejs-practicals/pullRequest?prid=9375994)
```suggestion
      .get("https://testapi.io/api/dartya/resource/cardata")
```